neps/nep-0463.md

```diff
@@ -0,0 +1,170 @@
+---
+  NEP: 463
+  Title: Decentralized Autonomous Organization Standard
+  Author: DAO Builders Group
+  Status: Draft
+  DiscussionsTo: nearbuilders.com/tg-dao
+  Type: Standards
+  Created: 21-Feb-2023
+---
+
+## Summary
+
+A standard interface to support decentralized autonomous organizations (DAOs) that enable proposals with governance.
+
+## Motivation
+
+There is a need for **interoperable DAO systems** on NEAR and other blockchains.
+
+Essentially, this NEP requests a review of the minimum viable Sputnik DAO.
+
+### Benefits:
+
+Standardizing the Sputnik DAO contract would provide:
+
+* greater accessibility
+* better coordination
+* easier development
+
+(compared to not having a standard interface)
+
+#### *What use cases does it support?*
+
+* *complete display of “My DAOs” in wallets and on-chain profiles*
+* *composability across platforms with a common ABI for any `DAO`*
+* *cohesive, yet decentralized governance by NEAR communities*
+
+DAO contract standards help builders create new DAOs that integrate with existing DAOs.
+
+## Rationale
+
+[Sputnik DAOs](https://github.com/near-daos/sputnik-dao-contract) represent a widely used contract system on NEAR, and the code has been thoroughly reviewed and tested in the community. Our hope is to improve accessibility of that robust open-source product by formally introducing a compatible standard. Ultimately, we are limiting complexity of the interface to include only necessary functions.
+
+## Specification 
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The `DAO` contract storage contains `Policy`, `Proposals`, and `Config`.
+
+### [Application Binary Interface](https://github.com/near-daos/sputnik-dao-contract/blob/main/sputnikdao2/ABI.md)
+
+```
+{
+  "viewMethods": [
+    "version",
+    "get_config",
+    "get_policy",
+    "get_staking_contract",
+    "has_blob",
+    "get_locked_storage_amount",
+    "get_available_amount",
+    "delegation_total_supply",
+    "delegation_balance_of",
+    "delegation_balance_ratio",
+    "get_last_proposal_id",
+    "get_proposals",
+    "get_proposal",
+    "get_bounty",
+    "get_last_bounty_id",
+    "get_bounties",
+    "get_bounty_claims",
+    "get_bounty_number_of_claims",
+    "get_factory_info"
+  ],
+  "changeMethods": [
+    "new",
+    "migrate",
+    "store_blob",
+    "remove_blob",
+    "add_proposal",
+    "act_proposal",
+    "bounty_claim",
+    "bounty_done",
+    "bounty_giveup",
+    "register_delegation",
+    "delegate",
+    "undelegate"
+  ],
```

Comment:
If the purpose of this NEP is to standardize interfaces, they have to also define the type of input arguments and return values.

Refer to the [NEP template section descriptions](https://github.com/near/NEPs/blob/master/nep-0000-template.md#specification) to learn the expectations. Given that this list of function names does not help to standardize the interface, I recommend removing it and expanding the "Interfaces" section to cover every single method you want to standardize.